admin-tools/src/webapp/vm/aliases/chef_aliases_edit.vm

```diff
@@ -10,7 +10,7 @@
 			<h3>
 				$tlang.getString("alias.alias")
 			</h3>	
-		#if ($alertMessage)<div class="alertMessage">$tlang.getFormattedMessage("alias.alert.message", $validator.escapeHtml($alertMessage))</div>#end
+		#if ($alertMessage)<div class="sak-banner-error">$tlang.getFormattedMessage("alias.alert.message", $validator.escapeHtml($alertMessage))</div>#end
```

Comment:
The "alias.alert.message" seems to take a replacement value, which doesn't seem to be used here. Maybe using "alias.alert" is what was intended (once it is restored in the previous file/comment).